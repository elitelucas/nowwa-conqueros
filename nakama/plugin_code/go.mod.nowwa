module plugin_code

go 1.14

require (
	github.com/golang/protobuf v1.4.2
	github.com/heroiclabs/nakama-common v1.8.0
	google.golang.org/protobuf v1.25.0
	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
)

replace github.com/golang/protobuf => github.com/golang/protobuf v1.4.2
replace google.golang.org/protobuf => google.golang.org/protobuf v1.25.0
