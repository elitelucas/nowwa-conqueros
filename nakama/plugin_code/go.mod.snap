module plugin_code

go 1.13

require (
    github.com/heroiclabs/nakama-common v1.0.0
    github.com/golang/protobuf v1.3.2
)

replace (
    github.com/golang/protobuf => github.com/golang/protobuf v1.3.2
)
